README.md

```diff
@@ -1,7 +1,9 @@
 Laravel Binput
 ==============
 
-Laravel Binput was created by, and is maintained by [Graham Campbell](https://github.com/GrahamCampbell), and is an input protector for [Laravel 5](http://laravel.com). It utilises my [Laravel Security](https://github.com/GrahamCampbell/Laravel-Security) package. Feel free to check out the [change log](CHANGELOG.md), [releases](https://github.com/GrahamCampbell/Laravel-Binput/releases), [license](LICENSE), and [contribution guidelines](CONTRIBUTING.md).
+Laravel Binput was created by, and is maintained by [Graham Campbell](https://github.com/GrahamCampbell), and is an input protector for [Laravel 5](http://laravel.com) that strips unwanted elements, e.g. `<script>` tags from any input you receive. It utilises my [Laravel Security](https://github.com/GrahamCampbell/Laravel-Security) package. 
```

Comment:
I might have misunderstood something. 😉 Please let me know what I should change.